Documentation/deployment.md

```diff
@@ -42,7 +42,7 @@ $ cd matchbox-v0.5.0-linux-amd64
 
 ## Install
 
-### RPM-based Distro
+### RPM-Based distro
```

Comment:
RPM-Based is a compound word, so capitalized it as per the common style guides (Chicago Manual of Style etc.)